content/helpers.md

```diff
@@ -0,0 +1,42 @@
++++
+title = "Helpers"
+draft = false
+gh_repo = "chef-web-docs"
+
+product = ["client", "workstation"]
+
+[menu]
+  [menu.infra]
+    title = "Custom Resource Helpers"
+    identifier = "chef_infra/resources/custom_resources/helpers Library Helpers"
+    parent = "chef_infra/resources/custom_resources"
+    weight = 60
++++
+
+Helper classes enable users to share code between Custom Resources. Rspec method test are possible when code is abstracted into plain Ruby files.
```

Comment:
Might be good to push people towards using resource partials here rather than helpers.

Reply:
Helpers are pretty terrible. Anything we could do here would be helpful.

Reply:
We need to scope a "document helpers" issue.